curvefs/src/tools/build/curvefs_build_topology_tool.cpp

```diff
@@ -42,8 +42,8 @@ void UpdateFlagsFromConf(curve::common::Configuration* conf) {
         LOG(INFO) << "conf: " << FLAGS_mds_addr;
     }
 
-    if (GetCommandLineFlagInfo("rpcTimeOutMs", &info) && info.is_default) {
-        conf->GetUInt32Value("rpcTimeoutMs", &FLAGS_rpcTimeOutMs);
+    if (GetCommandLineFlagInfo("rpcTimeOut", &info) && info.is_default) {
+        conf->GetUInt32Value("rpcTimeout", &FLAGS_rpcTimeOutMs);
```

Comment:
Why change to rpcTimeOut ? The conf file is rpcTimeoutMs

Reply:
> Why change to rpcTimeOut ? The conf file is rpcTimeoutMs

fix